<ui version="4.0" >
 <class>articleDialog</class>
 <widget class="QDialog" name="articleDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Article Viewer</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="8" >
    <widget class="QTextBrowser" name="textBrowser" />
   </item>
   <item row="1" column="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="6" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>410</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="6" colspan="2" >
    <widget class="QPushButton" name="closeButton" >
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="selectButton" >
     <property name="text" >
      <string>Select All</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="7" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5" colspan="2" >
    <widget class="QPushButton" name="saveButton" >
     <property name="text" >
      <string>Save...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="copyButton" >
     <property name="text" >
      <string>Copy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
